framework/CMakeLists.txt

```diff
@@ -532,6 +532,10 @@ if(${NEED_LINK_ATOMIC})
     target_link_libraries(${PROJECT_NAME} PUBLIC atomic)
 endif()
 
+# I'm uncertain why this has to be done explicitly, when the target_link_libraries call above should do it
```

Comment:
Is this comment still current? If so we should tackle the issue before merging.

Reply:
So if it's not possible to fix this, could you rephrase the comment to not refer to you personally (the "I'm..." part). (can't explain it better, not a native English speaker).

Reply:
it must have been some transient glitch.  I can't reproduce the error now.